tap_core/src/rav/request.rs

```diff
@@ -12,13 +12,13 @@ use crate::{
 
 /// Request to `tap_aggregator` to aggregate receipts into a Signed RAV.
 #[derive(Debug)]
-pub struct RAVRequest {
+pub struct RAVRequest<Rcpt> {
```

Comment:
```suggestion
pub struct RavRequest<Rcpt> {
```

?. You were considering similar changes elsewhere.